File: clustermanage-client-javafx/src/main/java/cc/bitky/clustermanage/netty/message/web/WebMsgOperateBoxUnlock.java
package cc.bitky.clustermanage.netty.message.web;

import cc.bitky.clustermanage.netty.message.MsgType;
import cc.bitky.clustermanage.netty.message.base.BaseMessage;

/**
 * 服务器操作远程开锁
 */
public class WebMsgOperateBoxUnlock extends BaseMessage {

    public WebMsgOperateBoxUnlock(int groupId, int boxId) {
        super(groupId, boxId);
        setMsgId(MsgType.SERVER_REMOTE_UNLOCK);
    }
}

File: Project-v1-Obsolete/hardware/SingleDeviceTestNET/SingleDeviceTestNET/message/WebMsgOperateBoxUnlock.cs
using SocketServer.message.@base;
using SocketServer.utils;

namespace SocketServer.message
{
    public class WebMsgOperateBoxUnlock : BaseMessage
    {
        public WebMsgOperateBoxUnlock(int groupId, int boxId) : base(groupId, boxId)
        {
            setMsgId(MsgType.ServerRemoteUnlock);
        }
    }
}